.npmignore

```diff
@@ -0,0 +1,3 @@
+/coverage/
+/node_modules/
+/test/
```

Comment:
Is this file necessary? We specify `"files"` in __package.json__.

Reply:
In that case no 😆